lighthouse-core/module/index.js

```diff
@@ -40,7 +53,7 @@ module.exports = function(url, flags, config) {
       config = defaultConfig;
     }
 
-    const driver = new ChromeProtocol();
+    const driver = new ChromeProtocol(traceCategories);
```

Comment:
let's set the categories onto the driver in `driver/index.js` inside of `setupDriver()`.